types/next-auth/client.d.ts

```diff
@@ -52,6 +52,7 @@ declare function signin(
         callbackUrl?: string;
         redirect?: boolean;
     },
+    authorizationParams?: GenericObject
```

Comment:
[In the source code](https://github.com/nextauthjs/next-auth/blob/main/src/client/index.js#L232) I see this ends passed to the `URLSearchParams()` constructor. What this constructor expects is actually a more mixed type than just an object; I think we should then update to 📐
```ts
authorizationParams?: string | string[][] | GenericObject | URLSearchParams
```

Reply:
```suggestion
    authorizationParams?: string | string[][] | GenericObject | URLSearchParams
```

Ohh, i didn't see that, thx!
So, we better add that then